### ISSUE TITLE
Successfully Flashed "KMC 4 Outlet WiFi"

### ISSUE THREAD

**Issue by jluzonpr** [ct-Open-Source/tuya-convert#91]
Had 2 of these from a few months ago, but they seem to no longer be in stock on Amazon.
https://www.amazon.com/gp/product/B0786B1LGM

That being said, succesful flash and configuration.

```
GPIO 13 Relay 2
GPIO 14 Relay 3
GPIO 15 Relay 1
GPIO 16 Button
```

Still gotta work on the Power Metering, but I had no idea how to do this part. Going to dig a bit but any help would be great.

GPIO4 and GPIO 5 numbers increase when set to Counter1 and Counter2 and something is set, but not sure what these would mean.

**Comment by chaseadam** [comment]
https://templates.blakadder.com/kmc_4.html

**Comment by NickJLange** [comment]
Can we confirm is OTA Update flashed?

**Comment by adobito** [comment]
@NickJLange I don't know this specific one, but I tried flashing some other KMC smart switches I bought recently with Tuya Convert and it didn't work.

**Comment by CSSMWX** [comment]
I got two of these October 2023 and it all worked great! 
OTA, klicking, templating, updates.
Mine were these
https://www.amazon.com/dp/B07SWZ2N4J
On back it says Model 30608
made 10/22
Type 3 SPD
VPR 800V (L-N)
They are nicely made, recommended.

**Comment by HiryuX** [comment]
@CSSMWX Am a total noob to this -- what do you use to apply OTA flashing on this device?  Thanks!

**Comment by hartzell** [comment]
Adding a note for posterity, I have several of these "KMC 30608" devices that i've flashed by taking them apart and using a jig to connect to power, tx, rx, and gpio 0.  The first two I did a while back (Jan/Feb 2024) and don't remember any difficulty.  Bought two more in Feb but didn't touch them until now (Sept).  I was able to flash them, but it took *dozens* of tries to get `esptool.py` to talk to them.  Not sure what was wonky (jig may have worn a bit, or ...).  Eventually they worked.

I'm using this template:

```
{"NAME":"KMC-30608","GPIO":[0,320,0,0,0,0,0,0,0,225,226,224,32,0],"FLAG":0,"BASE":18}
```
